src/templates/index.html

```diff
@@ -58,10 +58,13 @@ <h3 class="title">Hello World App</h3>
         </div>
         <div class="col-md-6">
           <h3 class="title">Easy To Start</h3>
-          <p>Let's create a minimal web app in 5 minutes using Python 3.</p>
+          <p>Let's create a minimal web app in 5 minutes.</p>
           <h4>1. Install Python</h4>
-          <p>Install the latest Python 3 from <a href="https://www.python.org/downloads/">Python.org</a>.</p>
+          <p>Install the latest Python from <a
+                  href="https://www.python.org/downloads/">Python.org</a>.
+          Works fine under Python 2 or 3.</p>

```

Comment:
Maybe just remove it? The previous amended statement implies the same thing anyway.